maps/reddit_comments.py

```diff
@@ -0,0 +1,44 @@
+import praw
+
+def scrape_reddit_comments(url):
+    # Initialize praw with your credentials (create an app on Reddit to get these)
+    reddit = praw.Reddit(client_id='g1B_Pdsfwszzw5xbUf3i3g',
```

Comment:
You committed your reddit secrets! Please use environment variables to pass these in.

Reply:
also, rotate these secrets.